### PR TITLE
add helm hub china in README.md

### PR DESCRIPTION
**Pull request by wonderflow**
add helm hub China in README.md

### CODE REVIEW

**Comment by wonderflow** [comment]
Please help review, Thanks! @michelleN 

\cc @resouer

**Comment by cpanato** [comment]
@wonderflow please sign the DCO

**Comment by wonderflow** [comment]
@cpanato done

**Review by cpanato — APPROVED** [review]
lgtm

**Comment by cpanato** [comment]
@mattfarina can you please take a look?

**Comment by wonderflow** [comment]
@mattfarina @michelleN please help take a look, Thanks!

**Comment by prydonius** [comment]
AFAIK we're not linking to hub.helm.sh in this README, should we add that here, or is this really the right place for this?

**Comment by resouer** [comment]
@prydonius Aha, yeah, actually there are two parts of work for Helm Hub China as well:

1. We replaced all unavailable image URLs like `gcr.io` in Charts of this repo and periodically sync them to `https://github.com/cloudnativeapp/charts`, which should be the right link for this README I think.
2. The "localized" Charts then would be visible on `https://developer.aliyun.com/hub`, which we will try to link it in other repo (`/hub`?) later.

The PR is updated, thanks for comment!

**Comment by wonderflow** [comment]
@prydonius  The link is updated and changed to https://github.com/cloudnativeapp/charts, thanks for comment.

\cc @resouer

**Comment by prydonius** [comment]
Thanks @resouer @wonderflow 

> The "localized" Charts then would be visible on https://developer.aliyun.com/hub, which we will try to link it in other repo (/hub, /monocular?) later.

+1 I think we can link to there from the hub repo

File: README.md [hunk 1]
```diff
@@ -99,3 +99,7 @@ To provide that support the API versions of objects should be those that work fo
 ## Status of the Project
 
 This project is still under active development, so you might run into [issues](https://github.com/helm/charts/issues). If you do, please don't be shy about letting us know, or better yet, contribute a fix or feature.
+
+## Happy helming in China
+
+If you are in China, there are some problems to use Helm Charts directly (e.g. gcr.io, googleapis.com etc), you can use this mirror repo at https://github.com/cloudnativeapp/charts.
```
Comment (resouer) [inline README.md:105]: maybe:

If you are in China, there are some problems to use upstream Helm charts (e.g. images hosted on gcr.io, quary.io, and charts hosted on googleapis.com etc), we recommend you to use the "localized" repo at https://github.com/cloudnativeapp/charts, which periodically sync charts from here and automatically replace unavailable URL by local mirror in every chart.

**Pull request merged**